tox.ini

```diff
@@ -32,14 +32,26 @@ deps = https://github.com/willkg/dennis/archive/aca57f6d.zip
 commands = dennis-cmd lint --errorsonly locale/
 
 [testenv:docker]
-commands = 
-    docker-compose -f docker-compose.yml -f scripts/docker-compose.travis.yml up -d
-    docker exec -i kuma_web_1 make compilejsi18n collectstatic
+whitelist_externals =
+    docker
+    docker-compose
+    make
+    sleep
+setenv =
+    COMPOSE_FILE = docker-compose.yml:scripts/docker-compose.travis.yml
+    COMPOSE_PROJECT_NAME = kumatox
+commands =
+    make build-base VERSION=latest
+    docker-compose build mysql
+    docker-compose up -d
+    sleep 5
+    docker-compose exec -T web make compilejsi18n collectstatic
     # TODO: mysql needs some time to set up. find a way to eliminate the waiting time
     sleep 60
-    docker logs kuma_mysql_1
-    docker exec -it kuma_web_1 ./manage.py migrate
-    docker exec -i kuma_web_1 make test
+    docker-compose logs mysql
+    docker-compose exec web ./manage.py migrate
+    docker-compose exec -T web make test
+    docker-compose stop
```

Comment:
It is required to run it when I run ``tox -e docker`` locally, or the  tox environment takes over the ports needed for development.  I could just do it manually, but it takes all of 5 seconds, so why not include it...